generators/svelte/templates/cypress/integration/entities/_entityFolder_/_entityFile_-list.spec.js.ejs

```diff
@@ -90,10 +90,13 @@ _%>)
         cy.wait(100)
 	})
 
+<%_ if (searchEngineAny) { _%>
```

Comment:
I didn’t create yet.

Reply:
ok. Instead of bypassing tests, I would skip use of this option at CI level and add a note that it's buggy in the JHipster8. Once JHipster fixes the issue, we just need to enable that option in CI. If someone still generates the application with this option, then, let that fail